neoload/commands/test_results.py

```diff
@@ -101,6 +101,39 @@ def junit(__id, junit_file):
     displayer.print_result_junit(json_result, json_sla_test, json_sla_interval, junit_file)
 
 
+def get_current_sla_data():
```

Comment:
This function seems never used and have an issue : Unresolved reference 'is_current_test_results_set'

Reply:
Removed